### PR TITLE
UG: add delClient command

### LINKED ISSUE #82: Add delClient section to UG

### PR DESCRIPTION
**Pull request by Thing1Thing2**
close #82

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -124,20 +124,19 @@ Examples:
...
-* The index refers to the index number shown in the displayed person list.
+* Deletes the client at the specified `INDEX`.
+* The index refers to the index number shown in the displayed clients' list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
-
+* `list` followed by `delClient 2` deletes the 2nd client in the address book.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:138]: Please follow the example format written by other people
change it to `delClient i/2`

**Review by ThomasHoooo — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:138]: okay :+1:

File: docs/UserGuide.md [hunk 2]
```diff
@@ -179,14 +151,14 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+MyEduRec data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+MyEduRec data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:158]: Should be MyInsuRec

File: docs/UserGuide.md [hunk 3]
```diff
@@ -179,14 +151,14 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+MyEduRec data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:154]: Should be MyInsuRec

File: docs/UserGuide.md [hunk 4]
```diff
@@ -179,14 +151,14 @@ Format: `exit`
...
 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+MyEduRec data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes its format invalid, MyEduRec will discard all data and start with an empty data file at the next run.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:161]: Should be MyInsuRec

File: docs/UserGuide.md [hunk 5]
```diff
@@ -77,66 +78,37 @@ Examples:
...
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Displays information about the client at the specific `INDEX`.
+* The index refers to the index number shown in the displayed clients' list.
+* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+* `listClient` followed by `viewClient i/1` displays information about the 1st client in MyInsuRec.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:98]: Please standardise format

File: docs/UserGuide.md [hunk 6]
```diff
@@ -77,66 +78,37 @@ Examples:
...
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+* Deletes the client at the specified `INDEX`.
+* The index refers to the index number shown in the displayed clients' list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `listClient` followed by `delClient i/2` deletes the 2nd client in MyInsuRec.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:111]: Please standardise format

File: docs/UserGuide.md [hunk 7]
```diff
@@ -163,13 +141,7 @@ Format: `delMeeting i/INDEX`
...
 
 Examples:
-* `listMeeting` followed by `delMeeting 2` deletes the 2nd Meeting in the meeting list.
-
-### Clearing all entries : `clear`
-
-Clears all entries from the address book.
-
-Format: `clear`
+* `listMeeting` followed by `delMeeting i/2` deletes the 2nd Meeting in the meeting list.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:144]: Please standardise format

**Review by ThomasHoooo — CHANGES_REQUESTED** [review]

**Review by ThomasHoooo — APPROVED** [review]

**Comment by ThomasHoooo** [comment]
LGTM

**Pull request merged**